api/v1beta1/openstackdataplanedeployment_types.go

```diff
@@ -41,10 +60,6 @@ type OpenStackDataPlaneDeploymentSpec struct {
 	// AnsibleSkipTags for ansible execution
 	// +kubebuilder:validation:Optional
 	AnsibleSkipTags string `json:"ansibleSkipTags,omitempty"`
-
-	// +kubebuilder:validation:Optional
-	// ServicesOverride list
-	ServicesOverride []string `json:"servicesOverride"`
```

Comment:
Dropping the concept of a servicesOverride means it's on the user to remember the list of services for each NodeSet. The only place that would be stored is on a previous deployment. If the user wants to deploy again, they have to manually use the same list.